### ISSUE TITLE
BUG - [Low] - [All] - Accessibility 5103 - Screenreaders incorrectly announce a sentence as a heading on "How to submit evidence" page.

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va-mobile-app#12218]
## What happened?

Sentence is incorrectly announced as a heading on "How to submit evidence" page.

WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.

Voiceover and Talkback announce this sentence (see screen shots below) as Heading.

## Specs:

- Device: iPhone, Android
- OS: iOS, Android.
- User Account: +228. (Cory Sohrakoff can set claims to need 5103 flow, if needed.)
- Accessibility State: Screen reader ON.

Slack thread for adding user data:  https://dsva.slack.com/archives/C09B6NPTBC4

## Steps to Reproduce

1. With screen reader on, Navigate to 5103 How to submit evidence page.
2. Read the sentence after the bulleted list beginning "Your notice about evidence we may need..."
3. Screen reader says this sentence is a heading.

## Desired behavior

Should just read the sentence, no announcement of heading.
## Acceptance Criteria

Reads sentence with no announcement of heading.
## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: Low 

- 3 - Low

## Linked to Story

Story:  https://github.com/department-of-veterans-affairs/va-mobile-app/issues/11993

PR:  https://github.com/department-of-veterans-affairs/va-mobile-app/pull/12205

## Screen shot(s) and additional information

<img width="666" height="767" alt="Image" src="https://github.com/user-attachments/assets/ea59579e-a6c7-43dd-9fb7-a15097691cae" />

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
